Src/WindowsServer/WindowsServer.Shared/FirstChanceExceptionStatisticsTelemetryModule.cs

```diff
@@ -220,13 +220,22 @@ private void TrackStatistics(string type, string operation, string method, bool
 
         private string GetDimCappedString(string value, ConcurrentDictionary<string, int> capValues)
```

Comment:
can you rename value to something like dimensionValue or something? value seem like a keyword...